nimare/meta/ibma.py

```diff
@@ -337,12 +358,15 @@ class Stouffers(IBMAEstimator):
 
     _required_inputs = {"z_maps": ("image", "z")}
 
-    def __init__(self, use_sample_size=False, **kwargs):
+    def __init__(self, use_sample_size=False, two_sided=True, **kwargs):
```

Comment:
this changes the default to concordant right? we should make note that this is different from previous behavior (but I do agree with changing this default behavior)